src/logic/operate.test.js

```diff
@@ -0,0 +1,19 @@
+import operate from './operate';
+
+describe('Test to operator function', () => {
+  test('First test operate', () => {
+    expect(operate('9', '3', '-')).toBe('6');
+  });
+
+  test('Second test operate', () => {
+    expect(operate('9', '3', '+')).toBe('12');
+  });
+
+  test('Third test operate', () => {
+    expect(operate('10', '2', '÷')).toBe('5');
+  });
+
+  test('Fourth test operate', () => {
+    expect(operate('9', '3', 'x')).toBe('27');
+  });
+});
```

Comment:
- [ ] Nice work testing most of the functionalities here. If you remember, one of the project requirements states that you should test all functionalities. Kindly add a test example for the _**modulus operator**_ `(%)`. 👍